deploy/src/main/resources-filtered/jobservice-deployment.yaml

```diff
@@ -144,6 +144,11 @@ spec:
               configMapKeyRef:
                 name: jobservice-config
                 key: CAF_WEBSERVICE_URL
+          - name: CAF_JOB_SERVICE_RESUME_JOB_QUEUE
```

Comment:
Not sure about the name `CAF_JOB_SERVICE_RESUME_JOB_QUEUE`. 

We have `CAF_WORKER_PAUSED_QUEUE` so maybe `CAF_JOB_SERVICE_RESUMED_QUEUE` might be a better name so that the two queues have a similar naming pattern, e.g. `<WORKER>_<ACTION>_QUEUE`

`CAF_WORKER_PAUSED_QUEUE`
`CAF_JOB_SERVICE_RESUMED_QUEUE`

What do you think?

Reply:
Put "request" into the queue name: `CAF_JOB_SERVICE_RESUME_REQUEST_QUEUE`.

Reply:
Or your original suggestion is fine: `CAF_JOB_SERVICE_RESUME_JOB_QUEUE`

Reply:
Thanks, I'll leave it as is then: `CAF_JOB_SERVICE_RESUME_JOB_QUEUE`